Examples/AppDelegate.m

```diff
@@ -19,7 +20,6 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     if ([[NSProcessInfo processInfo].arguments containsObject:@"useFastAnimations"]) {
         self.window.layer.speed = 100;
     }
-
```

Comment:
Nit: if we don’t need to substantively change this file, let’s also remove the white space change.